src/openrouter.py

```diff
@@ -20,13 +20,24 @@ async def llm_call(
         completion = await client.chat.completions.create(
             model=llm_model,
             temperature=0,
+            top_p=0,
+            seed=9999,
+            max_tokens=1000,
```

Comment:
please explain why we want this parameters to be set to such values?

Reply:
It's my fault. I left the old call structure in the branch, even though it hasn't been used in the local branch for a long time